src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,51 @@
+import { FC, useState, useEffect } from 'react';
+
+import { Person } from '../types';
+
+import { getPeople } from '../api';
+
+import { Loader } from '../components/Loader';
+import { PeopleTable } from '../components/PeopleTable';
+
+export const PeoplePage: FC = () => {
+  const [hasError, setHasError] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+  const [people, setPeople] = useState<Person[]>([]);
+
+  useEffect(() => {
+    setIsLoading(true);
+
+    getPeople()
+      .then(setPeople)
+      .catch(() => setHasError(true))
+      .finally(() => setIsLoading(false));
```

Comment:
The 'isLoading' state is set to true at the beginning of the useEffect hook and set to false at the end of the hook after the API call. However, if the 'getPeople' function fails and the catch block is executed, the 'isLoading' state will not be set to false. This would mean that the Loader component would be displayed indefinitely even after the API call has failed. A better approach would be to set 'isLoading' to false in the catch block as well as in the finally block.